setup.py

```diff
@@ -15,7 +15,7 @@
             'genesis-pyapi==1.2.1',
             'pyclipper==1.1.0.post1',
             'Orange3>=3.22.0',
-            'cellannotation',
+            'point-annotator',
```

Comment:
Can you specify version? Use [compatible-release option](https://www.python.org/dev/peps/pep-0440/#compatible-release) or something similar.

Thanks!